src/patternfly/demos/Card/card-view-demo-template-gallery.hbs

```diff
@@ -180,4 +180,17 @@
         Expose REST services and their APIs using Swagger specification.
       {{/card-body}}
     {{/card}}
+    {{#> card card--id="card-11" card--modifier="pf-m-compact pf-l-bullseye"}}
```

Comment:
While it looks like this works, we can't always rely on adding `.pf-l-bullseye` to an existing component, since there may be overlapping styles, a product may add the card styles after the bullseye styles, we may make an update to the card or bullseye that breaks the other component, etc. Also the bullseye layout is a separate component in react. So here we would just use `{{#> bullseye}}` as a child of `{{#> card}}`